src/ReactTwistWebpackPlugin.js

```diff
@@ -23,7 +23,7 @@ const DEFAULT_OPTIONS = {
     twistPlugin: '@twist/babel-plugin-transform-react',
 
     // Webpack-specific options:
-    babelLoaderTest: /\.jsx$/,
+    babelLoaderTest: /\.((tsx?)|(jsx))$/,
```

Comment:
probably should change to `jsx?` as well, if you're supporting .ts

Reply:
Agreed. Let me do that.